tests/candev/main.c

```diff
@@ -196,6 +202,12 @@ int main(void)
     puts("Initializing Linux Can device");
     candev_linux_init( &linux_dev, &(candev_linux_conf[0]));    /* vcan0 */
     candev = (candev_t *)&linux_dev;
+
+#elif  defined(MODULE_MCP2515)
```

Comment:
There is an extra space in this `elif`, do you  mind uncrustifying your files to check there is nothing else missing.